addons/web/static/src/js/views/basic/basic_model.js

```diff
@@ -3401,23 +3401,30 @@ var BasicModel = AbstractModel.extend({
         var self = this;
         var isValid = true;
         var element = this.localData[id];
+        var currentPage = self._applyX2ManyOperations(element).data;
         _.each(element._changes, function (command) {
             if (command.operation === 'DELETE' ||
                     command.operation === 'FORGET' ||
                     command.operation === 'REMOVE_ALL') {
                 return;
             }
-            var recordData = self.get(command.id, {raw: true}).data;
-            var record = self.localData[command.id];
-            _.each(element.getFieldNames(), function (fieldName) {
-                var field = element.fields[fieldName];
-                var fieldInfo = element.fieldsInfo[element.viewType][fieldName];
-                var rawModifiers = fieldInfo.modifiers || {};
-                var modifiers = self._evalModifiers(record, rawModifiers);
-                if (modifiers.required && !self._isFieldSet(recordData[fieldName], field.type)) {
-                    isValid = false;
-                }
-            });
+            var recordID = command.id;
+            // if the record is not on the page, we skip its validation
+            // because it may have not been fetched (except for its id/sequence)
+            // thus trying to validate such records would crash.
```

Comment:
If That line crashes, if it would have crashed anyway (see before the fix line 3410/3411).

Reply:
no, in comment you say 'would crash', and i suggest to replace it by 'could crash', since, as far as i understand, it only crashes if there are unknown fields to evaluate.